app/assets/stylesheets/navbar.css

```diff
@@ -16,12 +94,9 @@
     border-color: transparent;
     width: 100%;
 }
-.navbar-default .navbar-brand {
-    color: white;
-}
 
 .navbar-default .navbar-nav > li > a {
-    color: white;
+    color: white !important;
```

Comment:
I don't know if that fits here, but when checked out locally you have a white-colored font on white background
![image](https://user-images.githubusercontent.com/23058484/35004024-fdb50a74-faee-11e7-947d-cb2fd4ad4200.png)

Reply:
Works for me